### PR TITLE
[WIP] Fluxible Router

### PR DESCRIPTION
**Pull request by mridgway**
Relies on https://github.com/yahoo/flux-router-component/issues/41

### CODE REVIEW

**Comment by yahoocla** [comment]
CLA is valid!

File: fluxible-router/components/Home.js [hunk 1]
```diff
@@ -0,0 +1,13 @@
+/**
+ * Copyright 2014, Yahoo! Inc.
+ * Copyrights licensed under the New BSD License. See the accompanying LICENSE file for terms.
+ */
+import React from 'react';
+
+class About extends React.Component {
```
Comment (redonkulus) [inline fluxible-router/components/Home.js]: Home

File: fluxible-router/components/Home.js [hunk 2]
```diff
@@ -0,0 +1,13 @@
...
+    render() {
+        return <p>Welcome to the site!</p>;
+    }
+}
+
+export default About;
```
Comment (redonkulus) [inline fluxible-router/components/Home.js]: Home

**Comment by mridgway** [comment]
:+1:

**Pull request merged**